lesson9.tex

```diff
@@ -0,0 +1,123 @@
+\documentclass{lessonplan}
+
+% Constants
+\newcommand{\lessonNumber}{}
+\title{Lesson \lessonNumber: The B\&B Stock Exchange Game}
+\author{\linkHome}
+\date{}
+
+% Buildup
+\setLessonNumber{\lessonNumber}
+
+
+%Document
+\begin{document}
+
+  \maketitle
+
+  \section{Overview}
+    \subsection{Summary}
+      This lesson will be a simulation of a stock exchange on a tropical island named Bot Island. 
+      We will teach student what stocks are, how they work, and how they are influenced by real-world events.
+      For more advanced students, it delves into how the stocks make money (through compound interest and dividends)
+      and what are the best strategies for making the most money.
+
+    \subsection{Objectives}
+    \begin{itemize}
+      \item Learn what a stock is, and how to buy and sell stocks
+      \item (Intermediate) Learn about compound interest, and how money grows when you leave it in 
+                an account that gains interest. 
+      \item (Advanced) Learn different strategies of how to handle money regarding the market (such as 
+               diversification, investing in hot-item stocks, avoiding the stock market altogether, etc), and discuss 
+               different kinds of ways to diversify a portfolio (including but not limited to bonds, index funds, 
+               mutual funds, and ETFs). Also learn how to examine and rate a stock by using information like balance sheets P/E ratios.
+      \item (Cooperative) Students can create their own "firms" by grouping up and discussing how they would buy or sell their stocks.
```

Comment:
Quotations should use '' style notation. See here: https://www.maths.tcd.ie/~dwilkins/LaTeXPrimer/QuotDash.html#:~:text=Single%20quotation%20marks%20are%20produced,left%20quotation%20marks%20are%20required.)

Reply:
Done!